demos/motion_tracking/dvs_file_input/process.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+from numpy.lib.stride_tricks import as_strided
+from scipy import signal
+from dv import AedatFile
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import OutPort
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.model.py.ports import PyOutPort
+from lava.magma.core.resources import CPU
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.decorator import implements, requires
+
+
+
+class DVSFileInput(AbstractProcess):
```

Comment:
I am expecting that we will replace this custom Process with the `Inivation` Process, once that has been developed and merged into lava-peripherals. Please capture this in an issue.

Reply:
Done (#72)